client-v2/src/components/FrontsEdit/Edit.tsx

```diff
@@ -81,11 +97,21 @@ class FrontsEdit extends React.Component<Props> {
             id={frontsContainerId}
             makeRoomForExtraHeader={this.props.isCurrentFrontsMenuOpen}
           >
-            {this.props.frontIds.map(id => (
-              <SingleFrontContainer key={id} id={createFrontId(id)}>
-                <FrontContainer frontId={id} />
-              </SingleFrontContainer>
-            ))}
+            {this.props.frontIds.length ? (
+              this.props.frontIds.map(id => (
+                <SingleFrontContainer key={id} id={createFrontId(id)}>
+                  <FrontContainer frontId={id} />
+                </SingleFrontContainer>
+              ))
+            ) : (
+              <NoFrontContainer>
+                <span>
+                  Select a front with the{' '}
```

Comment:
I added some dummy text to the placeholder here @akemitakagi - feel free to remove / change it but given I had to add an element for when we have no fronts I thought I'd add a bit of sign posting!

<img width="1027" alt="Screenshot 2019-05-15 at 18 29 48" src="https://user-images.githubusercontent.com/1652187/57796098-76b92180-773f-11e9-86c7-9e2073357a84.png">

Reply:
mega 👍